common/djangoapps/student/tests/test_models.py

```diff
@@ -383,3 +384,20 @@ def test_retirement(self):
         self.assertFalse(ManualEnrollmentAudit.objects.filter(enrollment=enrollment).exclude(
             reason=""
         ))
+
+
+class TestAccountRecovery(TestCase):
+    """ Tests for the AccountRecovery Model """
+
+    def test_retire_recovery_email(self):
+        """ Assert that Account Record for a given user is deleted when `retire_recovery_email` is called """
+        # Create user and associated recovery email record
+        user = UserFactory()
+        AccountRecoveryFactory(user=user)
+        assert len(AccountRecovery.objects.filter(user_id=user.id)) > 0
```

Comment:
Nit: Since there should only ever be 1 row for a user, would it make more sense for this to be `== 1`?

Reply:
Good point - I'll make that change too